src/openzaak/utils/middleware.py

```diff
@@ -15,3 +33,72 @@ def __call__(self, request):
 
     def log(self, request: HttpRequest):
         logger.debug("Request headers for %s: %r", request.path, request.headers)
+
+
+class JWTAuth(_JWTAuth):
+    def __init__(self, encoded: str = None):
+        self.encoded = encoded
+        self.component = None
+
+    @property
+    def applicaties(self) -> Union[list, None]:
```

Comment:
maybe wouldn't override the entire property, but re-implement `def _request_auth(self)` so that it always returns an empty list?

Reply:
`_request_auth(self)` is changed, `applicaties` is inherited from the parent class